spark/src/main/scala/org/apache/comet/CometExecIterator.scala

```diff
@@ -75,12 +75,11 @@ class CometExecIterator(
       metricsUpdateInterval = COMET_METRICS_UPDATE_INTERVAL.get(),
       new CometTaskMemoryManager(id),
       batchSize = COMET_BATCH_SIZE.get(),
-      use_unified_memory_manager =
-        CometSparkSessionExtensions.cometUnifiedMemoryManagerEnabled(conf),
-      memory_pool_type = COMET_EXEC_MEMORY_POOL_TYPE.get(),
-      memory_limit = CometSparkSessionExtensions.getCometMemoryOverhead(conf),
-      memory_limit_per_task = getMemoryLimitPerTask(conf),
-      task_attempt_id = TaskContext.get().taskAttemptId,
+      offHeapMode = CometSparkSessionExtensions.isOffHeapEnabled(conf),
+      memoryPoolType = COMET_EXEC_MEMORY_POOL_TYPE.get(),
+      memoryLimit = CometSparkSessionExtensions.getCometMemoryOverhead(conf),
+      memoryLimitPerTask = getMemoryLimitPerTask(conf),
+      taskAttemptId = TaskContext.get().taskAttemptId,
```

Comment:
Drive-by cleanup to use camel case not snake case. Also renamed `use_unified_memory_manager` to `offHeapMode` because we are no longer using this to indicate when to use unified memory management. That is now decided in native code based on `offHeapMode` and `memoryPoolType`.